src/objects/entities/Entity.ts

```diff
@@ -133,8 +133,8 @@ class Entity extends Phaser.GameObjects.Container {
     ) as PhaserMatterImage;
     this.scene.add.existing(this);
 
-    if (this.collisionCategory) {
-      this.gameObject.setCollisionCategory(this.collisionCategory);
+    if (collisionCategory) {
```

Comment:
We could remove the entity class property `collisionCategory ` now then if we're not using it?

Reply:
yeah but it public, and i wasnt sure if it was used anywhere

i guess i need to check and report back

Reply:
yeah it's used here

https://github.com/ScottyXIII/bigheaded/blob/main/src/objects/entities/Ben3.ts#L21

so cannot be removed from being a class property